innsender/src/main/kotlin/no/nav/soknad/innsending/util/mapping/tilleggsstonad/TilleggsstonadJson2JsonConverter.kt

```diff
@@ -85,7 +85,14 @@ fun getMaalgruppeinformasjonFromLivssituasjon(
 ): JsonMaalgruppeinformasjon? {
 	// Bruk maalgruppeinformasjon hvis dette er hentet fra Arena og lagt inn på søknaden
 
-	if (livssituasjon == null) return null
+	if (livssituasjon == null)  {
+		/* Setter default målgruppe for å kunne sende inn ungdomsprogrammer */
+		return JsonMaalgruppeinformasjon(
```

Comment:
Nå vil alltid `maalgruppeinformasjon` settes dersom `livssituasjon` mangler, også for ikke-ungdomsprogram-søknadene.

Det er avklart at det er ok?